### ISSUE TITLE
Not an issue just thumbs up

### ISSUE THREAD

**Issue by 1N50MN14** [codekitchen/dinghy#36]
This project is too awesome, restored my faith in Docker and saved my Macbook from being killed by a varangian hammer.

I owe you beer!

**Comment by codekitchen** [comment]
You are too kind! Thanks, I'm glad it's useful.

**Comment by MrMMorris** [comment]
+1 to this. There were some days with vanilla B2D that I wanted to give up on Docker entirely!

**Comment by markoshust** [comment]
+1 here also. Working with Magento (lots of files), impossible without something like this

**Comment by markoshust** [comment]
Used dinghy for a new Magento 2 development process. Thought you'd like to know @codekitchen http://mageinferno.com/blog/magento-2-development-docker-os-x

**Comment by levinotik** [comment]
:+1: great work

**Comment by weilera** [comment]
:+1: indeed

**Comment by jakajancar** [comment]
:-1:

Perhaps the NFS (and DNS?) wizardry can one day be merged into the docker-machine xhyve driver, to make an ultimate replacement for VirtualBox/VMWare Fusion on OS X.

**Comment by Kallin** [comment]
:+1:  I really hope that the devs here are in touch with some devs from the docker project. I was really struggling with docker and ember-cli, couldn't get file system notification reloads to work, then stumbled across this project and now w/ nfs shares it works great!

**Comment by assembledadam** [comment]
Spent hours trying to find a solution for Docker on OSX with Virtualbox shared folders being shit.

Attempted getting NFS working on Boot2Docker - no go. Tried sshfs shares, that was just broken. Tried Rsync with fswatch - great for small projects, but for those with many files, pretty useless and time consuming.

It's now past 1am and I'm very glad to have found this project.  Within 30 mins my dev setup appears to be working perfectly, just the small matter of getting a wrapper script to set it up for other devs with no effort.

:+1:

**Comment by damonomad** [comment]
> restored my faith in Docker

that about sums it up for me too. Thank you.

**Comment by dcosson** [comment]
:+1:

**Comment by zwily** [comment]
:+1:

**Comment by Babacooll** [comment]
A project that took over 90 seconds to fully load now takes 5 seconds. Thank you ! Very good job !

**Comment by mickaelperrin** [comment]
Just thanks for this project. I switched from Linux to Mac OS X and was thinking about to go back before trying dinghy. It works so far so good... Apple should sponsor you :) :+1:

**Comment by ryansch** [comment]
I agree.  I had spent time getting Parallels shared folders working with RancherOS only to find out that the performance was terrible (although not as bad as virtualbox).  After finding out that somehow NFS is still the state of the art for VM file sharing, I discovered dinghy while looking for solutions to the inotify issues.  Dinghy neatly solved all of those issues and is the basis for our development environment.  Thanks!!

**Comment by NeoPhi** [comment]
Just wanted to say that using Dinghy with VirtualBox has been great. I struggled with filesystem events not being properly propagated while using Docker for Mac but have not had a single issue with Dinghy. Kudos!

**Comment by assembledadam** [comment]
Been more than two years since my previous comment, and still using this project every day. Just wanted to pour some more praise on here since nobody has posted in a while!
